README.md

```diff
@@ -7,107 +7,20 @@
 
 istio-csr is an agent that allows for [Istio](https://istio.io) workload and
 control plane components to be secured using
-[cert-manager](https://cert-manager.io). Certificates facilitating mTLS, inter
-and intra cluster, will be signed, delivered and renewed using [cert-manager
+[cert-manager](https://cert-manager.io). Certificates facilitating mTLS - both inter
+and intra cluster - will be signed, delivered and renewed using [cert-manager
 issuers](https://cert-manager.io/docs/concepts/issuer).
 
-⚠️ Currently supports Istio versions v1.7+
-
-⚠️ Currently supports cert-manager versions v1.3+
+istio-csr supports Istio versions v1.7+ and cert-manager v1.3+
 
 ---
 
```

Comment:
self review after internal weekly meeting: README needs some high-level information for power uers

Also for most it'll likely be really hard in practice to install istio-csr _after_ istio, so we can also mention that in the README